### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by puspakhanal70726**
Hi, bro check the code

### CODE REVIEW

File: src/main/java/hello/Application.java [hunk 1]
```diff
@@ -33,3 +33,8 @@ public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
     }
 
 }
+\\
```
Comment (mahato861) [inline src/main/java/hello/Application.java:36]: good

File: src/main/java/hello/Application.java [hunk 2]
```diff
@@ -33,3 +33,8 @@ public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
...
+\\\
+\\\
+\
```
Comment (mahato861) [inline src/main/java/hello/Application.java:39]: best

File: src/main/java/hello/Application.java [hunk 3]
```diff
@@ -33,3 +33,8 @@ public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
     }
 
 }
+\\
+\\\
```
Comment (mahato861) [inline src/main/java/hello/Application.java:37]: better

**Review by mahato861 — COMMENTED** [review]
hello bro i checked ur code it's all look good ready to release

**Comment by puspakhanal70726** [comment]
made the chage

**Comment by mahato861** [comment]
ok go ahed

**Pull request merged**